packages/grafana-data/src/dataframe/__snapshots__/ArrowDataFrame.test.ts.snap

```diff
@@ -265,7 +265,7 @@ Object {
       "labels": undefined,
       "name": "float32_values",
       "type": "number",
-      "values": Array [
+      "values": Float32Array [
```

Comment:
NOTE:  the real test is that the rest of the null values in this file that are still supported :)